src/languages/php.js

```diff
@@ -313,7 +314,7 @@ export default function(hljs) {
           regex.concat(WHITESPACE, "+"),
           // to prevent built ins from being confused as the class constructor call
           regex.concat("(?!", normalizeKeywords(BUILT_INS).join("\\b|"), "\\b)"),
-          /\\?\w+/,
+          regex.concat("\\\\?", IDENT_RE, "+"),
```

Comment:
Idea already includes `[at least one][more*]`... why do we need `+`?

Reply:
@joshgoebel Fixed